CHANGELOG.md

```diff
@@ -16,6 +16,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   during sign-in.
 - Added ip access control based on the `allow_access_from` field of `Account`
   during sign-in.
+- Added the `language` GraphQL API to get the user's UI language selection, and
+  the `updateLanguage` GraphQL API to modify it. The existing `insertAccount`
+  and `updateAccount` APIs required specific user roles, so we introduced these
+  new APIs to allow users to change their UI language settings regardless of
+  their role.
```

Comment:
Since the thing about user roles is not a new information and users can check its detail in the code, I don't think it would be better to remove the second sentence.

Reply:
As you pointed out, I removed the second sentence.